tailwind.config.js

```diff
@@ -1,42 +1,76 @@
-const plugin = require("tailwindcss/plugin");
-const { fontFamily } = require("tailwindcss/defaultTheme");
-
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  darkMode: ["class"],
   content: [
-    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
-  ],
+    './pages/**/*.{ts,tsx}',
```

Comment:
Actualmente todo está en la carpeta `src`

Creo que basados en: https://nextjs.org/docs/app/building-your-application/styling/tailwind-css#configuring-tailwind

Podríamos simplemente agregar: `    './src/**/*.{js,ts,jsx,tsx,mdx}',`

Reply:
(Noté que esto fue hecho por `shadcn`, pero creo que está roto en nuestro proyecto por nuestra configuración :)